### PR TITLE
Add GtkSourceView 5.0

### PR DESCRIPTION
**Pull request by taozuhong**
Add GtkSourceView 5 for GTK4

### CODE REVIEW

File: mingw-w64-gtksourceview5/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "../${_realname}-${pkgver}"
+
+  ${MINGW_PREFIX}/bin/meson.exe compile
+}
+
+package() {
+  cd "${srcdir}/build-${MINGW_CHOST}"
+  DESTDIR="${pkgdir}" ${MINGW_PREFIX}/bin/meson.exe install
+
+  install -Dm644 "${srcdir}/${_realname}-${pkgver}/COPYING" "${pkgdir}${MINGW_PREFIX}/share/licenses/${_realname}4/COPYING"
```
Comment (Biswa96) [inline mingw-w64-gtksourceview5/PKGBUILD:46]:
```suggestion
  install -Dm644 "${srcdir}/${_realname}-${pkgver}/COPYING" "${pkgdir}${MINGW_PREFIX}/share/licenses/${_realname}5/COPYING"
```

**Comment by Biswa96** [comment]
1. Maintainer should be your name/alias.
2. gcc and ninja can be skipped in make depends as those are obvious.
3. vala is missing.

File: mingw-w64-gtksourceview5/PKGBUILD (hunk 1 above)
Comment (taozuhong) [reply mingw-w64-gtksourceview5/PKGBUILD]: Got it, thanks.

**Comment by taozuhong** [comment]
before submitted PR, I had got passed the build in my machine.

**Comment by Biswa96** [comment]
1. Maintainer should be your name/alias.
2. gcc and ninja can be skipped in make depends as those are obvious.
3. vala is missing in make depends. 👈

**Comment by mati865** [comment]
`ninja` is not included in https://packages.msys2.org/group/mingw-w64-x86_64-toolchain so it's necessary.

**Comment by Biswa96** [comment]
ninja is now dependency of meson, so ninja exists when meson does https://packages.msys2.org/package/mingw-w64-x86_64-meson?repo=mingw64

**Comment by lazka** [comment]
(mingw32 will fail until gtk4 rebuild is in staging, so ignore for now)

**Pull request merged**